README.md

```diff
@@ -1 +1,55 @@
-# SWE_RLJ
\ No newline at end of file
+# Factor Calculator Project
+
+## Table of Contents
+1. [Project Description](#project-description)
+2. [Example Input](#example-input)
+3. [Example Output](#example-output)
+4. [System Diagram](#system-diagram)
+5. [Prototype Code](#prototype-code)
+
+---
+
+## Project Description
+The Factor Calculator is a program designed to process a CSV file containing integers in its first row. For each integer, the program computes its factors and generates an output CSV file where each integer's factors are placed in the rows below each input integer.
+
+### Factors Explained
```

Comment:
Change finding factors of a number to finding the factorial of a number instead.

Reply:
got it. will redo